src/components/LabeledRadioButton/LabeledRadioButton.jsx

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+import RadioButton from '../RadioButton/RadioButton';
+
+const boundClassNames = bindClassNames('LabeledRadioButton');
+const {
+	bool,
+	func,
+	node,
+	object,
+	string,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
+ *
+ * This is a composite of the `RadioButton` component and the native `label`
+ * element.
+ */
+const LabeledRadioButton = React.createClass({
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates whether the component should appear and act disabled by
+		 * having a "greyed out" palette and ignoring user interactions.
+		 */
+		isDisabled: bool,
+
+		/**
+		 * Indicates that the component is in the "selected" state when true
+		 * and in the "unselected" state when false.
+		 */
+		isSelected: bool,
+
+		/**
+		 * Used to identify the purpose of this radio button to the user -- can
+		 * be any renderable content.
+		 */
+		label: node,
```

Comment:
There isn't a `Label` component at the moment... just a plain old regular prop. :)